src/sdJwt/disclosureMapping.ts

```diff
@@ -0,0 +1,242 @@
+// This file contains helpers functions for mapping between disclosures entries and the payload of an SD-JWT.
```

Comment:
I can do this later, but this would benefit from some comments :). Well, the whole repo could benefit from that..

Reply:
I'll add some before we merge. Better to do it now than later